.gitignore

```diff
@@ -9,3 +9,4 @@
 /build/x86
 /test/docker-test/*[.html, .log, .pyc]
 .vscode
+.idea
```

Comment:
why?

Reply:
it was just part of the original commit
it is just .gitignore rule to ignore some IDE generated files i think